.github/workflows/notify-mm-blog.yml

```diff
@@ -17,12 +17,11 @@ jobs:
         shell: bash
         run: |
           .github/workflows/notify-mm-blog.sh
-      - name: Notify channel 1
+      - name: Notify "Community Configuration" channel on community
         uses: mattermost/action-mattermost-notify@master
         env:
```

Comment:
This should be `with` instead of `env`. Also might need to define `PAYLOAD`. See new usage for v2.0/master - https://github.com/mattermost/action-mattermost-notify?tab=readme-ov-file#usage